homeassistant/components/squeezebox/media_player.py

```diff
@@ -196,12 +192,6 @@ async def _discovered_player(player):
         },
         "async_call_query",
     )
-    platform.async_register_entity_service(
```

Comment:
Can we keep them around for 2 releases, log a warning and then forward the call to `async_join_players` ?

Reply:
👍